### PR TITLE
 release controller installer job s390x

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

**Comment by deepsm007** [comment]
Depends on https://github.com/openshift/release/pull/20190

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.8.yaml [hunk 1]
```diff
@@ -216,6 +216,17 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/serial
     workflow: openshift-e2e-openstack-ipi
+<<<<<<< HEAD
```
Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.8.yaml:219]: looks like rebase was incorrect

Comment (LakshmiRavichandran1) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.8.yaml]: fixed them in the latest commit

**Review by deepsm007 — APPROVED** [review]

**Pull request merged**